version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // VERSION ...
-const VERSION = "0.11.1"
+const VERSION = "0.12.0"
```

Comment:
why minor update? only the readme has changed: https://github.com/bitrise-io/stepman/compare/0.11.1...master